bidwire/base_notifier.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+import sendgrid
+from sendgrid.helpers.mail import *
+from yattag import Doc
+
+ADMIN_EMAIL = "bidwire-admin@googlegroups.com"
+DEBUG_EMAIL = "bidwire-logs@googlegroups.com"
+
+log = logging.getLogger(__name__)
+
+
+class BaseNotifier:
+    def __init__(self, type):
+        self.type = type
+
+    def get_items(self, bid):
+        return None
```

Comment:
Let's make this and the other unimplemented methods in the `Base*` classes at least throw a `NotImplementedException`, to make sure they don't get accidentally used.

An alternative to throwing would be to go down the full AbstractBaseClass route, with a partial base implementation, which I think would be ideal, but might be too much change or this PR. Up to you. (More on Python's ABC: https://docs.python.org/3.6/library/abc.html)